libs/components/forms/src/assets/locales/resources_en_US.json

```diff
@@ -7,10 +7,6 @@
     "_description": "Screen reader only label for character count over limit symbol",
     "message": "You are over the character limit."
   },
-  "skyux_checkbox_group_required": {
```

Comment:
I'm curious about the name change, here. Will this get used by another component?

Reply:
Yes I'm planning to use this for radio group and field group

Reply:
Cool.